helm/istio/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.global.multitenant }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
is the goal here to make it so that only pods in the mesh can talk to other pods in the mesh? Would this break any migration strategies? People can modify this, but is it too tight by default?

Reply:
The goal is to ensure all members of the mesh can talk to all other members and to the control plane.  This is effectively white listing communication between them.  It is not restricting any other communication (i.e. it doesn't black list anything).  Users are free to constrain their projects however they want.  That said, if they wanted to constrain what services were available to the mesh, they should create a separate project for those and expose them to mesh member, e.g. by giving specific access to the private pod from a member pod/namespace.  (This is similar to policy optimization, which you can read about [here](https://docs.openshift.com/container-platform/3.9/admin_guide/managing_networking.html#admin-guide-networking-using-networkpolicy-efficiently).